### DISCUSSION TITLE
Is it possible to configure moonraker-telegram-bot in this mod

### DISCUSSION THREAD

**Discussion by ag-set** [xblax/flashforge_ad5m_klipper_mod#236] [Ideas]
Will there be enough free resources to run moonraker-telegram-bot. How to configure it?

**Comment by consp** [comment]
Doesn't seem too big, you can always try it out yourself to see the memory footprint. This will not be plug an play though.

You will need to make startup scripts yourself since systemd is not running.
The install script will _NOT_ work. Manual setup is required.

**Comment by fernplant** [comment]
I setup Google Chat notifications. Seems to work but have only run it for 2 days so far so cannot confirm long term yet
